share/spice/untappd/footer.handlebars

```diff
@@ -0,0 +1,7 @@
+<div class="untappd__footer tile__update__info two-line">
+    <span>{{abv}}% ABV</span>
+    <i class="sep"></i>
+    <span>{{ibu}} IBU</span>
+    <i class="sep"></i>
+    <span><i class="count__i"></i> {{ratings}}</span>
```

Comment:
There is one thing because of which I have not done this already. The default limit for untappd API is 100 calls per hour per access key. It's a pretty small number, I guess we can't let ourselves produce up to 26 requests for each IA.